sdk/scheduler/src/main/java/com/mesosphere/sdk/framework/OfferProcessor.java

```diff
@@ -29,6 +29,7 @@
 import com.mesosphere.sdk.scheduler.Metrics;
 import com.mesosphere.sdk.scheduler.OfferResources;
 import com.mesosphere.sdk.scheduler.MesosEventClient.OfferResponse;
+import com.mesosphere.sdk.scheduler.MesosEventClient.StatusResponse;
```

Comment:
Does `ClientStatusResponse` make sense?

It just feels weird to have 1 differentiated StatusResponse and then the undifferentiated StatusResponse that's for a specific purpose.

Reply:
SGTM, will rename now while fixing conflicts

Reply:
👍